.devops/code-review-pipelines.yml

```
@@ -12,7 +12,7 @@ resources:
     - repository: pagopaEcommerceLocal
       type: github
       name: pagopa/pagopa-ecommerce-local 
-      ref: main
+      ref: CHK-3992-patch-auth-request-v2
```

```suggestion
      ref: main
```

TODO: to replace after dependent pr merge